docs/lib/auth/fragments/js/emailpassword.md

```diff
@@ -99,6 +99,13 @@ async function resendConfirmationCode() {
 
 ## Sign-out
 
+[Amazon Cognito now supports token revocation](https://aws.amazon.com/about-aws/whats-new/2021/06/amazon-cognito-now-supports-targeted-sign-out-through-refresh-token-revocation/)  and Amplify (from version 4.1.0) will revoke Amazon Cognito tokens if the application is online. This means Cognito refresh token cannot be used anymore to generate new Access and Id Tokens.
+
+Access and Id Tokens are short live (60 minutes is default but can be set from 5 minutes to 1 day). After revocation this tokens cannot be use with Cognito
+service anymore, however they are still valid when used with other services like AppSync or API Gateway.
```

Comment:
For limiting subsequent calls to these other services after invalidating tokens, we recommend lowering token expiration time for your app client in the Cognito User Pools console. If you are using the Amplify CLI this can be accessed by running `amplify console auth`.